presto-orc/src/main/java/io/prestosql/orc/reader/BooleanStreamReader.java

```diff
@@ -37,6 +36,7 @@
 import static io.prestosql.orc.metadata.Stream.StreamKind.DATA;
 import static io.prestosql.orc.metadata.Stream.StreamKind.PRESENT;
 import static io.prestosql.orc.stream.MissingInputStreamSource.missingStreamSource;
+import static io.prestosql.spi.block.RunLengthEncodedBlock.create;
```

Comment:
I'd avoid a static import for this. "create" is a very generic word, so it makes the code harder to understand.